sidebar_translations.json

```
@@ -37,6 +37,8 @@
   "develop.blocks.block_entities": "Block Entities",
   "develop.blocks.block_entity_renderer": "Block Entity Renderers",
   "develop.entities": "Entities",
+  "develop.entities.damage-types": "Damage Types",
+  "develop.entities.custom-entities": "Custom Entities",
```

```suggestion
  "develop.entities.custom_entities": "Custom Entities",
```

**Damage Types** is below